synapse/app/_base.py

```diff
@@ -115,6 +116,7 @@ def start_reactor(
 
     def run():
         logger.info("Running")
+        setup_jemalloc_stats()
```

Comment:
I changed it to get called here, mainly so that we try to set it up *after* we've got logging set up.